ADAL/src/public/ADTokenCacheItem.h

```diff
@@ -55,28 +55,30 @@
     NSMutableDictionary* _additionalClient;
 }
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Sorry for the back-and-forth here, wouldn't it be best to remove this line as well? Force devs to define the nullability if we add anything instead of having an assumption foisted on them and they weren't aware of.